src/transaction/log_manager.c

```diff
@@ -11360,7 +11365,7 @@ cdc_get_recdes (THREAD_ENTRY * thread_p, LOG_LSA * undo_lsa, RECDES * undo_recde
 	      {
```

Comment:
`undo_length = undo->length;` doesn't seem to be needed. The same goes for all in `if (undo_lsa != NULL)`.

Reply:
Thanks, I will check.

Reply:
done.